lib/gestures.dart

```diff
@@ -103,3 +103,7 @@ mixin ScaleDetector on Game {
 mixin MouseMovementDetector on Game {
   void onMouseMove(PointerHoverEvent event) {}
 }
+
+mixin ScrollDetector on Game {
```

Comment:
I think we should name this `MouseScrollDetector`, What do you think?

Reply:
Assuming it only triggers via mouse, and not mobile touch scroll, yes.

Reply:
@hastebrot can you confirm this? So we can merge this?